src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -515,7 +515,7 @@ private void CompileNamedType(NamedTypeSymbol containingType)
 
                     case SymbolKind.Property:
                         {
-                            SourcePropertySymbol sourceProperty = member as SourcePropertySymbol;
+                            var sourceProperty = member as SourcePropertySymbolBase;
```

Comment:
>SourcePropertySymbolBase [](start = 59, length = 24)

I assume you checked all other similar places where we do a cast or a type check for SourcePropertySymbol. Is this correct?

Reply:
Yup, I did a search for every place we used SourcePropertySymbol